### DISCUSSION TITLE
on refresh clear localstorage .

### DISCUSSION THREAD

**Discussion by Sidhota54** [saleor/saleor#11059] [Deployment]
login & authenticate working nicely on react-storefront .  when i refresh local storage clear or authenticate false .
on refresh the page again need to login.
plz help how to fix this error ?

**Comment by peelar** [comment]
Hello @Sidhota54 👋 

Please [file an issue](https://github.com/saleor/react-storefront/issues/new?assignees=&labels=type:+bug&template=bug.yml) in the react-storefront repository. Make sure to fill in all the necessary steps, that makes it easier for us to reproduce. Thank you 🙏.
